frontend/package.json

```diff
@@ -86,7 +86,7 @@
     "e2e": "jest --config jest/e2e.config.js",
     "e2e-localProd": "TEST_ENV=localProd jest --config jest/e2e.config.js",
     "e2e-localProd-ci": "DEBUG=pw:api TEST_ENV=localProd jest --config jest/e2e.config.js",
-    "e2e-dev": "TEST_ENV=dev jest --config jest/e2e.config.js",
```

Comment:
There is no connected TEST_ENV in `frontend/tests/common/constants.ts` should we point the test env to a URL or delete the e2e option entirely?

Reply:
Remove e2e-test